### PR TITLE
Fixed Default propery issue for the col-xs-* classes 

### PR DESCRIPTION
**Pull request by cnviradiya**
### Related issues

https://github.com/DivanteLtd/vue-storefront/issues/2550

### Short description and why it's useful

If I'm talk short regarding this issue then there are missing the default property for the `col-xs` in the default theme css `/vue-storefront/src/themes/default/css/vendor/_flexboxgrid2.scss`. and Also I seen there are write the code for the all the `col` class default css but It's for th jus start with `col` class So, It will not apply here. In below section I've mentioned the appropriate screenshort.
NOTE: This change is not affect 

### Screenshots of visual changes before/after (if there are any)

1. Home page grid
   Before: http://www.screencast.com/t/eXvvJKsPb
   After: http://www.screencast.com/t/vruJniOVGzz
2. Registration Popup
   Before: http://www.screencast.com/t/7Gmvp95UjhB
   After: http://www.screencast.com/t/V2V3ilebw
3. Checkout page personal details section
   Before: http://www.screencast.com/t/v2LKEQvAvp
   After: http://www.screencast.com/t/riiSrJxm9o

### Screenshot of passed e2e tests (if you are using our standard setup as a backend)

I'm comparing issue with your running live url `https://demo.vuestorefront.io`. and Installed in my local with the use of your backend.

### Upgrade Notes and Changelog

- [x] No upgrade steps required (100% backward compatibility)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/doc/Upgrade%20notes.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing VS sites with this new feature

### Contribution and currently important rules acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)
- [x] I read the [TypeScript Action Plan](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/TypeScript%20Action%20Plan.md) and adjusted my PR according to it
- [x] I read about [Vue Storefront Modules](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/api-modules/about-modules.md) and I am aware that every new feature should be a module

### CODE REVIEW

**Comment by filrak** [comment]
Are you sure global `birder-box` is not affecting other parts of the shop? Have you checked mobile version aswell?

**Comment by cnviradiya** [comment]
@filrak Yes, I'm sure and checked through the shop It's not affecting other parts. 
Yes, I've also checked in the mobile version and it's better looks. Let me share here screenshot for mobile version too.

Even better look then previous and closes all issue related alignments.

**Ref issue fixed**: https://github.com/DivanteLtd/vue-storefront/issues/2556
Before: https://nimb.ws/cE3Oj6
After: https://nimb.ws/vP8Lai


Here is the mobile responsive issue also fixed and its not reflecting the other parts of shop. Please have a look

**Home page Here section**
Before: https://nimb.ws/cERPyu
After: https://nimb.ws/YJ3U2P

**Registration page**
Before: https://nimb.ws/KEZsUS
After: https://nimb.ws/2msEzb

And most of the part this issue is fixed through the shop and I'm sure it's not reflecting other parts.

File: src/themes/default/css/vendor/_flexboxgrid2.scss [hunk 1]
```diff
@@ -83,6 +83,40 @@
   max-width: 100%;
 }
 
+.col-xs,
+.col-xs-1,
```
Comment (lukeromanowicz) [inline src/themes/default/css/vendor/_flexboxgrid2.scss]: probably could be simplified to 
```
[class^="col-xs"], [class*=" col-xs"] {
...
}
```

Comment (cnviradiya) [reply src/themes/default/css/vendor/_flexboxgrid2.scss]: @lukeromanowicz I think  below is better its works for both case 
```
[class*=" col-xs"] {
...
}
```

No need to write
`[class^="col-xs"]`

Comment (lukeromanowicz) [reply src/themes/default/css/vendor/_flexboxgrid2.scss]: Actually, there is a difference in matching. Please, take a look at https://jsfiddle.net/ycrLtgmf/

Comment (cnviradiya) [reply src/themes/default/css/vendor/_flexboxgrid2.scss]: @lukeromanowicz 
Yes, Got it difference.
Let me a simplified code as per your suggestion.
Thanks.

**Comment by patzick** [comment]
@cnviradiya as it is a bug on current demo - could you provide this PR to branch `hotfix/v1.8.4`? Thanks! :)

**Comment by cnviradiya** [comment]
> hotfix/v1.8.4

@patzick You mean to generate new pr with `hotfix/v1.8.4`  this branch, Am I right?

**Comment by patzick** [comment]
@cnviradiya no, then i'd just change base for this PR to hotfix :) branch for hotfix has to be created from hotfix branch (or master if there is no current hotfix branch) and PR back to this branch :)

**Comment by cnviradiya** [comment]
> @cnviradiya no, then i'd just change base for this PR to hotfix :) branch for hotfix has to be created from hotfix branch (or master if there is no current hotfix branch) and PR back to this branch :)

Okay, I've changed my base branch from `develop` to `hotfix/v1.8.4`

**Comment by patzick** [comment]
Close in favour of #2589

**Pull request closed without merging**